articles/hints_tips/UsefulTips/UsefulTipsApiArticle.md

```diff
@@ -0,0 +1,253 @@
+# How to implement the Studio Useful Tips service
+
+The `RwsAppStore.UsefulTips.Service` is a service provider for updating the Useful Tips that are displayed in Trados Studio 2019+.
```

Comment:
Please use <var:ProductName> instead of direct mention to Trados Studio 2019+ so we can dynamically change product name in the future if the case arises.